src/Core/Domain/Feature/ValueObject/FeatureId.php

```diff
@@ -61,9 +61,9 @@ public function getValue()
      *
      * @throws InvalidFeatureIdException
      */
-    private function assertIntegerIsGreaterThanZero($featureId)
+    private function assertIntegerIsGreaterThanZero($featureId): void
     {
-        if (!is_numeric($featureId) || 0 > $featureId) {
+        if (!is_int($featureId) || 0 > $featureId) {
```

Comment:
May use validator helper ?

Reply:
ValueObject shouldn't have dependencies. It's why we don't use `Validate` in VO.